src/front-end/src/components/MenteeForm.js

```diff
@@ -2,23 +2,54 @@ import React, { Component } from 'react';
 import { Form, Button, Input } from 'antd';
 import TimezonePicker from 'react-bootstrap-timezone-picker';
 import 'react-bootstrap-timezone-picker/dist/react-bootstrap-timezone-picker.min.css';
-import { marshallMenteeInfo, pushToDB } from '../helper-methods';
+import emailjs from 'emailjs-com';
+// import { marshallMenteeInfo, pushToDB } from '../helper-methods';
```

Comment:
Minor: please remove legacy code instead of commenting it. I'll do this one.

Reply:
Thank you!